build_deploy.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+set -exv
+
+IMAGE_NAME="quay.io/cloudservices/compliance-backend"
+IMAGE_TAG=$(git rev-parse --short=7 HEAD)
+
+# Replace before merging
+#if [[ -z "$QUAY_USER" || -z "$QUAY_TOKEN" ]]; then
+if [[ ! ( -z "$QUAY_USER" || -z "$QUAY_TOKEN" ) ]]; then
+    echo "QUAY_USER and QUAY_TOKEN must be set"
+    exit 1
+fi
+
+DOCKER_CONF="$PWD/.docker"
+mkdir -p "$DOCKER_CONF"
+
+#remove before merging
+podman build -f new.Dockerfile -t "${IMAGE_NAME}:${IMAGE_TAG}" .
```

Comment:
Oh, this needs to be updated

Reply:
Yes I'll remove all of this before removing the "WIP" prefix